kb/migrating-visibility-data-from-es6.md

```diff
@@ -0,0 +1,86 @@
+---
+slug: migrate-visibility-data-from-es6
+title: Migrate visibility data from ES6
+tags:
+  - kb-article
+date: 2022-29-08T00:00:00Z
+---
+
+We added support for Elasticsearch v7 (ES7) in the v1.7.0 update to Temporal Server.
+ES7 introduces several breaking changes, including the removal of mapping types.
+These changes make ES6 incompatible with ES7.
+
+You can choose from various ways to update Temporal Server to support ES7.
+Upgrade ES6 to ES7, and then use one of the following methods to update your server.
+
+Visibility data can be migrated through reindexing.
+After updating Elasticsearch, run a reindexing process to move data from ES6 to ES7.
```

Comment:
```suggestion
Upgrade ES6 to ES7, and then use one of the following methods to update your Temporal Server.
```

RE:
> Visibility data can be migrated through reindexing.
> After updating Elasticsearch, run a reindexing process to move data from ES6 to ES7.

Move this to the "Upgrade" section below - obviously one does not migrate data if there is a fresh installation.
So I think it makes sense to move those sentences into a place where it helps support the narrative.